### PR TITLE
Enh/1/branch a

### LINKED ISSUE #1: Todoissue
### **Exercises**

Ensure you have SSH access to Tiger GitHub. This  allows you to work without having to type in your username/password constantly.

Create a private repository named mle-training with an empty README.

Create a GitHub issue for the current assignment and add in the details based on what is requested in this assignment and assign it to yourself. Note the issue-id. 

Create a new branch (enh/<issue-id>/<short-msg>) and copy the [code-snippet here](https://drive.google.com/drive/folders/1ebqWUF06mkLRpmV-AjAjPMYgmh0cuUvX) and commit as-is including the README. Use appropriate commit messages as per the guidelines.

Add a .gitignore file to the repository to avoid committing unnecessary files like pyc files and the CSV data files in your current folder.  

Use Conda to set up a python development environment named mle-dev to run the script. Export the environments file as env.yml and commit it to the repository.

Update the README file with instructions on how to run the code.

Create a Pull Request (PR). Make sure the PR follows the expected standards. [Link the PR to the GitHub issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to ensure the issue is closed automatically when the PR is merged.

Add the facilitators for your session as collaborators on the mle-training repository and request a PR.

Once the PR is approved, merge the PR to master branch and [give it a tag ](https://git-scm.com/book/en/v2/Git-Basics-Tagging)(named version) `v0.1` and push that to the remote.

**Deliverable**

Submit the URL of the GitHub PR. Ensure that the submitted code in the PR works as advertised in the README.

### PR DESCRIPTION
**Pull request by Shivam-Tiger**
Completed all the mentioned task in Assignment 1.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# mle-training
\ No newline at end of file
+# mle-training
+To run the code in nonstandardcode.py :
+1. You need to setup the environment with the environment.yml file provided 
+2. After setting up the environment you can run "python3 nonstandardcode.py" in the console
```
Comment (sibashisc) [inline README.md]: Incomplete README. Add the following

1. Project description
2. Command to create environment from env.yml you generated
3. Command to activate environment
4. Command to run Python script
Add the above points in respective sections.

Always use syntax highlighting while specifying commands

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  - sqlite=3.38.5=hc218d9a_0
+  - threadpoolctl=2.2.0=pyh0d69192_0
+  - tk=8.6.12=h1ccaba5_0
+  - tornado=6.1=py310h7f8727e_0
+  - tzdata=2022a=hda174b7_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xz=5.2.5=h7f8727e_1
+  - zlib=1.2.12=h7f8727e_2
+  - zstd=1.5.2=ha4553b6_0
+prefix: /home/shivamgupta/miniconda3/envs/mle-dev
```
Comment (sibashisc) [inline environment.yml]: Remove prefix, this is system specific

File: nonstandardcode.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
+import numpy as np
```
Comment (sibashisc) [inline nonstandardcode.py:1]: Add a screenshot of the output from running this script in mle-dev environment

**Review by sibashisc — CHANGES_REQUESTED** [review]

**Review by sibashisc — APPROVED** [review]
Please use syntax highlighting when specifying commands

**Comment by Shivam-Tiger** [comment]
> Please use syntax highlighting when specifying commands

Noted

**Pull request merged**